tools/MVP/templates/View.cpp.in

```diff
@@ -0,0 +1,43 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "View.h"
+
+#include "Presenter.h"
+
+#include <string>
+
+#include <QVBoxLayout>
+#include <QLabel>
+#include <QPushButton>
+#include <QString>
+
+
+View::View(QWidget *parent) : QWidget(parent), m_presenter() {
+  auto layout = new QVBoxLayout();
+  auto button = new QPushButton("Increment", this);
+  m_label = new QLabel("0", this);
+
+  layout->addWidget(button);
+  layout->addWidget(m_label);
+
+  setLayout(layout);
+
+  connect(button, SIGNAL(clicked()), this, SLOT(notifyButtonClicked()));
```

Comment:
I know that it is not used much at the moment in Mantid, but should we try to enforce the usage of new signal/slot syntax in this example case?
https://wiki.qt.io/New_Signal_Slot_Syntax